tests/test_tensor/test_op.py

```diff
@@ -53,11 +53,11 @@ def test_linear():
 
     # torch.nn.functional.linear(torch.randn(1, in_dim), sharded_weight, sharded_bias)
     out = fc(input_tensor)
-    loss = out.sum()
```

Comment:
How will we support torch tensor's functionality? Implement all of them in the ColoTensor?

Reply:
Maybe, implement all of them in ColoTensor.